src/app/[locale]/settings/navbar/page.tsx

```diff
@@ -0,0 +1,47 @@
+import NavService from '@/services/navService';
+import EditCategoryForm from './EditCategoryForm';
+import EditItemForm from './EditItemForm';
+import Divider from '@/components/Divider/Divider';
+import AddItemForm from './AddItemForm';
+import AddCategoryForm from './AddCategoryForm';
+import React from 'react';
+
+export default async function Page() {
+  const stats = await NavService.get();
+
+  return (
+    <main>
+      <title>Kontrollpanel - Navigering</title>
```

Comment:
```suggestion
      <title>Control Panel - Navigation</title>
```
Use English to be consistent with the rest of the page.

Reply:
This does however seam to be a problem for all admin pages